### ISSUE TITLE
Envmap Update OnEnabled Inconsistency

### ISSUE THREAD

**Issue by Jammie3d** [Facepunch/sbox-issues#6350]
### Describe the bug

I have a scene with a dynamic envmap that's set to update OnEnabled.
When you first open the scene the lighting will be dim, if you toggle "Render Dynamically" off and on or move the envmap then it will become brighter.

When you press play it goes back to the dim lighting, but when you set the envmap to update each frame and press play it's bright.

I've also experienced the play mode lighting being different to the lighting when first opening the scene. In that case the play mode lighting looked more correct, video shown below.

### To Reproduce

1. Download [my_project.zip](https://github.com/user-attachments/files/16830618/my_project.zip)
2. Open scenes/minimal.scene
3. Toggle "Render Dynamically" on the Envmap Probe

### Expected behavior

The OnEnabled lighting should be similar to the other update strategies, and toggling it should give consistent results.

### Media/Files

Repro Project: [my_project.zip](https://github.com/user-attachments/files/16830618/my_project.zip)

Video Example: 

https://github.com/user-attachments/assets/fa1be6e8-107a-4bbe-9f26-eaccce60e588

**Comment by aylaylay** [comment]
The repro project doesn't include what's shown in your video?

**Comment by Jammie3d** [comment]
Yeah I tried reproducing that one in a new project but I couldn't for some reason, I'll try again and just send the whole project if I can't.

**Comment by Jammie3d** [comment]
The scene that gets loaded when you first open the editor seems to matter.

If you open this project and it loads up minimal.scene for you, switching to bathroom.scene will show the same behavior as the video.

If you open the project and bathroom.scene gets loaded first then the lighting will match how it is in play mode.

[my_project2.zip](https://github.com/user-attachments/files/16873903/my_project2.zip)

**Comment by aylaylay** [comment]
The envmaps in this repro are just totally fucked for me

https://github.com/user-attachments/assets/fcdcfbbe-f50f-47a1-812a-6abd1dd7c00a

**Comment by aylaylay** [comment]
@samzanemesis

**Comment by Nolankicks** [comment]
I get this too. It happens in some projects but not others.

**Comment by samzanemesis** [comment]
OnEnabled takes only the first sample capture, other strategies accumulate light bounces over time, so the light brightness is correct behavior, I could add an option to specify a number of samples in that case

In your scene, it's being captured before the map has been loaded, they should be updated when a map is loaded, I'll get it done

Regarding the fucked flickering, I'll boot up my AMD card and see

**Comment by Jammie3d** [comment]
Multi bounce is working great, thanks!

However, when changing scenes in the editor it seems like the captures still arent being updated.
